Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:alpine
```

Comment:
Given it's a Go app, let's use `golang` to build and the `scratch` base to run the app (multi-stage build).